### ISSUE TITLE
Unable to update / Commits behind

### ISSUE THREAD

**Issue by tjenvy** [SickChill/SickChill#5269]
### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master / 51328d1fb61ebc9704934e3cf14425453196790d
OS: macOS 10.11.6
Browser: Safari
What you did:See below
What happened:see below
What you expected:see below
Logs:
```
Multiple log parts see below
```

I am having a plethora of issues.  Firstly, I have been running sickchil, cloned form the zip form day one.  I have never received any commit updates till today, then its 117 behind.  Simple enough, update, restarts and fails.  I check the logs and there are errors with "overwriting files"
So I grabbed the latest pull, expanded it, run it.  Still 117 commits behind.  Scour the system no traces left, re-unpack, sane thing.  

logs after trying to update::

09:11:06 INFO::ThreadPoolExecutor-0_0 :: Config backup in progress...
09:11:06 INFO::ThreadPoolExecutor-0_0 :: Config backup successful, updating...
09:11:10 ERROR::ThreadPoolExecutor-0_0 :: [51328d1] git pull -f origin master returned : From https://github.com/SickChill/SickChill
 * branch            master     -> FETCH_HEAD
error: The following untracked working tree files would be overwritten by merge:
	contrib/images/background.jpg
	contrib/images/banner.xcf
	contrib/images/icon.png
	contrib/images/icon.xcf
	contrib/images/logo.xcf
	contrib/images/poster.xcf
	contrib/images/vector.bmp
	contrib/images/vector.svg
	contrib/images/vector.xcf
	contrib/images/vector2.bmp
	gui/slick/images/ico/android-chrome-144x144.png
	gui/slick/images/ico/android-chrome-192x192.png
	gui/slick/images/ico/android-chrome-256x256.png
	gui/slick/images/ico/android-chrome-36x36.png
	gui/slick/images/ico/android-chrome-384x384.png
	gui/slick/images/ico/android-chrome-48x48.png
	gui/slick/images/ico/android-chrome-512x512.png
	gui/slick/images/ico/android-chrome-72x72.png
	gui/slick/images/ico/android-chrome-96x96.png
	gui/slick/images/ico/apple-touch-icon-114x114.png
	gui/slick/images/ico/apple-touch-icon-120x120.png
	gui/slick/images/ico/apple-touch-icon-144x144.png
	gui/slick/images/ico/apple-touch-icon-152x152.png
	gui/slick/images/ico/apple-touch-icon-180x180.png
	gui/slick/images/ico/apple-touch-icon-57x57.png
	gui/slick/images/ico/apple-touch-icon-60x60.png
	gui/slick/images/ico/apple-touch-icon-72x72.png
	gui/slick/images/ico/apple-touch-icon-76x76.png
	gui/slick/images/ico/apple-touch-icon.png
	gui/slick/images/ico/browserconfig.xml
	gui/slick/images/ico/favicon-16x16.png
	gui/slick/images/ico/favicon-32x32.png
	gui/slick/images/ico/mstile-144x144.png
	gui/slick/images/ico/mstile-150x150.png
	gui/slick/images/ico/mstile-310x150.png
	gui/slick/images/ico/mstile-310x310.png
	gui/slick/images/ico/mstile-70x70.png
	gui/slick/images/ico/site.webmanifest
	gui/slick/images/network/youtube premium.png
	gui/slick/images/network/youtube red.png
	gui/slick/images/providers/magnetdl.png
	gui/slick/images/sickchill-sc.png
	gui/slick/images/sickchill.png
	gui/slick/images/subtitles/subtitulamos.png
	gui/slick/images/subtitles/tusubtitulo.png
	lib/bin/slugify
	lib/certifi-2018.10.15.dist-info/DESCRIPTION.rst
	lib/certifi-2018.10.15.dist-info/INSTALLER
	lib/certifi-2018.10.15.dist-info/LICENSE.txt
	lib/certifi-2018.10.15.dist-info/METADATA
	lib/certifi-2018.10.15.dist-info/RECORD
	lib/certifi-2018.10.15.dist-info/WHEEL
	lib/certifi-2018.10.15.dist-info/metadata.json
	lib/certifi-2018.10.15.dist-info/top_level.txt
	lib/cfscrape-1.9.5.dist-info/INSTALLER
	lib/cfscrape-1.9.5.dist-info/LICENSE
	lib/cfscrape-1.9.5.dist-info/METADATA
	lib/cfscrape-1.9.5.dist-info/RECORD
	lib/cfscrape-1.9.5.dist-info/WHEEL
	lib/cfscrape-1.9.5.dist-info/top_level.txt
	lib/chardet-3.0.4.dist-info/DESCRIPTION.rst
	lib/chardet-3.0.4.dist-info/INSTALLER
	lib/chardet-3.0.4.dist-info/METADATA
	lib/chardet-3.0.4.dist-info/RECORD
	lib/chardet-3.0.4.dist-info/WHEEL
	lib/chardet-3.0.4.dist-info/entry_points.txt
	lib/chardet-3.0.4.dist-info/metadata.json
	lib/chardet-3.0.4.dist-info/top_level.txt
	lib/enzyme-0.4.1.dist-info/INSTALLER
	lib/enzyme-0.4.1.dist-info/METADATA
	lib/enzyme-0.4.1.dist-info/RECORD
	lib/enzyme-0.4.1.dist-info/WHEEL
	lib/enzyme-0.4.1.dist-info/top_level.txt
	lib/idna-2.7.dist-info/INSTALLER
	lib/idna-2.7.dist-info/LICENSE.txt
	lib/idna-2.7.dist-info/METADATA
	lib/idna-2.7.dist-info/RECORD
	lib/idna-2.7.dist-info/WHEEL
	lib/idna-2.7.dist-info/top_level.txt
	lib/rarfile-3.0.dist-info/INSTALLER
	lib/rarfile-3.0.dist-info/METADATA
	lib/rarfile-3.0.dist-info/RECORD
	lib/rarfile-3.0.dist-info/WHEEL
	lib/rarfile-3.0.dist-info/top_level.txt
	lib/requests-2.20.0.dist-info/DESCRIPTION.rst
	lib/requests-2.20.0.dist-info/INSTALLER
	lib/requests-2.20.0.dist-info/LICENSE.txt
	lib/requests-2.20.0.dist-info/METADATA
	lib/requests-2.20.0.dist-info/RECORD
	lib/requests-2.20.0.dist-info/WHEEL
	lib/requests-2.20.0.dist-info/metadata.json
	lib/requests-2.20.0.dist-info/top_level.txt
	lib/slugify-0.0.1.dist-info/INSTALLER
	lib/slugify-0.0.1.dist-info/METADATA
	lib/slugify-0.0.1.dist-info/RECORD
	lib/slugify-0.0.1.dist-info/WHEEL
	lib/slugify-0.0.1.dist-info/top_level.txt
	lib/slugify.py
	lib/urllib3/contrib/_appengine_environ.py
	lib/urllib3/util/queue.py
	lib/urllib3-1.24.dist-info/INSTALLER
	lib/urllib3-1.24.dist
Aborting
Updating 51328d1..abc6aab
None
09:11:10 ERROR::ThreadPoolExecutor-0_0 :: [51328d1] Failed doing webui callback: Traceback (most recent call last):
  File "/Applications/SickRage/sickbeard/webserve.py", line 287, in async_call
  File "/Applications/SickRage/sickbeard/webserve.py", line 1299, in update
    # do a hard restart
  File "/Applications/SickRage/sickbeard/webserve.py", line 692, in _genericMessage
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'
Traceback (most recent call last):
  File "/Applications/SickRage/sickbeard/webserve.py", line 287, in async_call
    
  File "/Applications/SickRage/sickbeard/webserve.py", line 1299, in update
    # do a hard restart
  File "/Applications/SickRage/sickbeard/webserve.py", line 692, in _genericMessage
    
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'
09:11:10 ERROR::Uncaught exception in write_error
Traceback (most recent call last):
  File "/Applications/SickRage/lib/tornado/web.py", line 1036, in send_error
    self.write_error(status_code, **kwargs)
  File "/Applications/SickRage/sickbeard/webserve.py", line 192, in write_error
    t = PageTemplate(rh=self, filename="404.mako")
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'
09:11:17 INFO::ThreadPoolExecutor-0_1 :: Config backup in progress...
09:11:17 INFO::ThreadPoolExecutor-0_1 :: Config backup successful, updating...
09:11:20 ERROR::ThreadPoolExecutor-0_1 :: [51328d1] git pull -f origin master returned : From https://github.com/SickChill/SickChill
 * branch            master     -> FETCH_HEAD
error: The following untracked working tree files would be overwritten by merge:
	contrib/images/background.jpg
	contrib/images/banner.xcf
	contrib/images/icon.png
	contrib/images/icon.xcf
	contrib/images/logo.xcf
	contrib/images/poster.xcf
	contrib/images/vector.bmp
	contrib/images/vector.svg
	contrib/images/vector.xcf
	contrib/images/vector2.bmp
	gui/slick/images/ico/android-chrome-144x144.png
	gui/slick/images/ico/android-chrome-192x192.png
	gui/slick/images/ico/android-chrome-256x256.png
	gui/slick/images/ico/android-chrome-36x36.png
	gui/slick/images/ico/android-chrome-384x384.png
	gui/slick/images/ico/android-chrome-48x48.png
	gui/slick/images/ico/android-chrome-512x512.png
	gui/slick/images/ico/android-chrome-72x72.png
	gui/slick/images/ico/android-chrome-96x96.png
	gui/slick/images/ico/apple-touch-icon-114x114.png
	gui/slick/images/ico/apple-touch-icon-120x120.png
	gui/slick/images/ico/apple-touch-icon-144x144.png
	gui/slick/images/ico/apple-touch-icon-152x152.png
	gui/slick/images/ico/apple-touch-icon-180x180.png
	gui/slick/images/ico/apple-touch-icon-57x57.png
	gui/slick/images/ico/apple-touch-icon-60x60.png
	gui/slick/images/ico/apple-touch-icon-72x72.png
	gui/slick/images/ico/apple-touch-icon-76x76.png
	gui/slick/images/ico/apple-touch-icon.png
	gui/slick/images/ico/browserconfig.xml
	gui/slick/images/ico/favicon-16x16.png
	gui/slick/images/ico/favicon-32x32.png
	gui/slick/images/ico/mstile-144x144.png
	gui/slick/images/ico/mstile-150x150.png
	gui/slick/images/ico/mstile-310x150.png
	gui/slick/images/ico/mstile-310x310.png
	gui/slick/images/ico/mstile-70x70.png
	gui/slick/images/ico/site.webmanifest
	gui/slick/images/network/youtube premium.png
	gui/slick/images/network/youtube red.png
	gui/slick/images/providers/magnetdl.png
	gui/slick/images/sickchill-sc.png
	gui/slick/images/sickchill.png
	gui/slick/images/subtitles/subtitulamos.png
	gui/slick/images/subtitles/tusubtitulo.png
	lib/bin/slugify
	lib/certifi-2018.10.15.dist-info/DESCRIPTION.rst
	lib/certifi-2018.10.15.dist-info/INSTALLER
	lib/certifi-2018.10.15.dist-info/LICENSE.txt
	lib/certifi-2018.10.15.dist-info/METADATA
	lib/certifi-2018.10.15.dist-info/RECORD
	lib/certifi-2018.10.15.dist-info/WHEEL
	lib/certifi-2018.10.15.dist-info/metadata.json
	lib/certifi-2018.10.15.dist-info/top_level.txt
	lib/cfscrape-1.9.5.dist-info/INSTALLER
	lib/cfscrape-1.9.5.dist-info/LICENSE
	lib/cfscrape-1.9.5.dist-info/METADATA
	lib/cfscrape-1.9.5.dist-info/RECORD
	lib/cfscrape-1.9.5.dist-info/WHEEL
	lib/cfscrape-1.9.5.dist-info/top_level.txt
	lib/chardet-3.0.4.dist-info/DESCRIPTION.rst
	lib/chardet-3.0.4.dist-info/INSTALLER
	lib/chardet-3.0.4.dist-info/METADATA
	lib/chardet-3.0.4.dist-info/RECORD
	lib/chardet-3.0.4.dist-info/WHEEL
	lib/chardet-3.0.4.dist-info/entry_points.txt
	lib/chardet-3.0.4.dist-info/metadata.json
	lib/chardet-3.0.4.dist-info/top_level.txt
	lib/enzyme-0.4.1.dist-info/INSTALLER
	lib/enzyme-0.4.1.dist-info/METADATA
	lib/enzyme-0.4.1.dist-info/RECORD
	lib/enzyme-0.4.1.dist-info/WHEEL
	lib/enzyme-0.4.1.dist-info/top_level.txt
	lib/idna-2.7.dist-info/INSTALLER
	lib/idna-2.7.dist-info/LICENSE.txt
	lib/idna-2.7.dist-info/METADATA
	lib/idna-2.7.dist-info/RECORD
	lib/idna-2.7.dist-info/WHEEL
	lib/idna-2.7.dist-info/top_level.txt
	lib/rarfile-3.0.dist-info/INSTALLER
	lib/rarfile-3.0.dist-info/METADATA
	lib/rarfile-3.0.dist-info/RECORD
	lib/rarfile-3.0.dist-info/WHEEL
	lib/rarfile-3.0.dist-info/top_level.txt
	lib/requests-2.20.0.dist-info/DESCRIPTION.rst
	lib/requests-2.20.0.dist-info/INSTALLER
	lib/requests-2.20.0.dist-info/LICENSE.txt
	lib/requests-2.20.0.dist-info/METADATA
	lib/requests-2.20.0.dist-info/RECORD
	lib/requests-2.20.0.dist-info/WHEEL
	lib/requests-2.20.0.dist-info/metadata.json
	lib/requests-2.20.0.dist-info/top_level.txt
	lib/slugify-0.0.1.dist-info/INSTALLER
	lib/slugify-0.0.1.dist-info/METADATA
	lib/slugify-0.0.1.dist-info/RECORD
	lib/slugify-0.0.1.dist-info/WHEEL
	lib/slugify-0.0.1.dist-info/top_level.txt
	lib/slugify.py
	lib/urllib3/contrib/_appengine_environ.py
	lib/urllib3/util/queue.py
	lib/urllib3-1.24.dist-info/INSTALLER
	lib/urllib3-1.24.dist
Aborting

Running As Sudo (just as a test)   Same errors

09:13:22 ERROR::ThreadPoolExecutor-0_0 :: [51328d1] git pull -f origin master returned : From https://github.com/SickChill/SickChill
 * branch            master     -> FETCH_HEAD
error: The following untracked working tree files would be overwritten by merge:
	contrib/images/background.jpg
	contrib/images/banner.xcf
	contrib/images/icon.png
	contrib/images/icon.xcf
	contrib/images/logo.xcf
	contrib/images/poster.xcf
	contrib/images/vector.bmp
	contrib/images/vector.svg
	contrib/images/vector.xcf
	contrib/images/vector2.bmp
	gui/slick/images/ico/android-chrome-144x144.png
	gui/slick/images/ico/android-chrome-192x192.png
	gui/slick/images/ico/android-chrome-256x256.png
	gui/slick/images/ico/android-chrome-36x36.png
	gui/slick/images/ico/android-chrome-384x384.png
	gui/slick/images/ico/android-chrome-48x48.png
	gui/slick/images/ico/android-chrome-512x512.png
	gui/slick/images/ico/android-chrome-72x72.png
	gui/slick/images/ico/android-chrome-96x96.png
	gui/slick/images/ico/apple-touch-icon-114x114.png
	gui/slick/images/ico/apple-touch-icon-120x120.png
	gui/slick/images/ico/apple-touch-icon-144x144.png
	gui/slick/images/ico/apple-touch-icon-152x152.png
	gui/slick/images/ico/apple-touch-icon-180x180.png
	gui/slick/images/ico/apple-touch-icon-57x57.png
	gui/slick/images/ico/apple-touch-icon-60x60.png
	gui/slick/images/ico/apple-touch-icon-72x72.png
	gui/slick/images/ico/apple-touch-icon-76x76.png
	gui/slick/images/ico/apple-touch-icon.png
	gui/slick/images/ico/browserconfig.xml
	gui/slick/images/ico/favicon-16x16.png
	gui/slick/images/ico/favicon-32x32.png
	gui/slick/images/ico/mstile-144x144.png
	gui/slick/images/ico/mstile-150x150.png
	gui/slick/images/ico/mstile-310x150.png
	gui/slick/images/ico/mstile-310x310.png
	gui/slick/images/ico/mstile-70x70.png
	gui/slick/images/ico/site.webmanifest
	gui/slick/images/network/youtube premium.png
	gui/slick/images/network/youtube red.png
	gui/slick/images/providers/magnetdl.png
	gui/slick/images/sickchill-sc.png
	gui/slick/images/sickchill.png
	gui/slick/images/subtitles/subtitulamos.png
	gui/slick/images/subtitles/tusubtitulo.png
	lib/bin/slugify
	lib/certifi-2018.10.15.dist-info/DESCRIPTION.rst
	lib/certifi-2018.10.15.dist-info/INSTALLER
	lib/certifi-2018.10.15.dist-info/LICENSE.txt
	lib/certifi-2018.10.15.dist-info/METADATA
	lib/certifi-2018.10.15.dist-info/RECORD
	lib/certifi-2018.10.15.dist-info/WHEEL
	lib/certifi-2018.10.15.dist-info/metadata.json
	lib/certifi-2018.10.15.dist-info/top_level.txt
	lib/cfscrape-1.9.5.dist-info/INSTALLER
	lib/cfscrape-1.9.5.dist-info/LICENSE
	lib/cfscrape-1.9.5.dist-info/METADATA
	lib/cfscrape-1.9.5.dist-info/RECORD
	lib/cfscrape-1.9.5.dist-info/WHEEL
	lib/cfscrape-1.9.5.dist-info/top_level.txt
	lib/chardet-3.0.4.dist-info/DESCRIPTION.rst
	lib/chardet-3.0.4.dist-info/INSTALLER
	lib/chardet-3.0.4.dist-info/METADATA
	lib/chardet-3.0.4.dist-info/RECORD
	lib/chardet-3.0.4.dist-info/WHEEL
	lib/chardet-3.0.4.dist-info/entry_points.txt
	lib/chardet-3.0.4.dist-info/metadata.json
	lib/chardet-3.0.4.dist-info/top_level.txt
	lib/enzyme-0.4.1.dist-info/INSTALLER
	lib/enzyme-0.4.1.dist-info/METADATA
	lib/enzyme-0.4.1.dist-info/RECORD
	lib/enzyme-0.4.1.dist-info/WHEEL
	lib/enzyme-0.4.1.dist-info/top_level.txt
	lib/idna-2.7.dist-info/INSTALLER
	lib/idna-2.7.dist-info/LICENSE.txt
	lib/idna-2.7.dist-info/METADATA
	lib/idna-2.7.dist-info/RECORD
	lib/idna-2.7.dist-info/WHEEL
	lib/idna-2.7.dist-info/top_level.txt
	lib/rarfile-3.0.dist-info/INSTALLER
	lib/rarfile-3.0.dist-info/METADATA
	lib/rarfile-3.0.dist-info/RECORD
	lib/rarfile-3.0.dist-info/WHEEL
	lib/rarfile-3.0.dist-info/top_level.txt
	lib/requests-2.20.0.dist-info/DESCRIPTION.rst
	lib/requests-2.20.0.dist-info/INSTALLER
	lib/requests-2.20.0.dist-info/LICENSE.txt
	lib/requests-2.20.0.dist-info/METADATA
	lib/requests-2.20.0.dist-info/RECORD
	lib/requests-2.20.0.dist-info/WHEEL
	lib/requests-2.20.0.dist-info/metadata.json
	lib/requests-2.20.0.dist-info/top_level.txt
	lib/slugify-0.0.1.dist-info/INSTALLER
	lib/slugify-0.0.1.dist-info/METADATA
	lib/slugify-0.0.1.dist-info/RECORD
	lib/slugify-0.0.1.dist-info/WHEEL
	lib/slugify-0.0.1.dist-info/top_level.txt
	lib/slugify.py
	lib/urllib3/contrib/_appengine_environ.py
	lib/urllib3/util/queue.py
	lib/urllib3-1.24.dist-info/INSTALLER
	lib/urllib3-1.24.dist
Aborting
Updating 51328d1..abc6aab
None
09:13:23 INFO::MAIN :: SSL Error requesting url: 'https://sickchill.github.io/sb_network_timezones/network_timezones.txt' You have OpenSSL 0.9.8zh 14 Jan 2016, try upgrading OpenSSL to 1.0.1e+
09:13:23 INFO::MAIN :: SSL Error requesting url: 'https://sickchill.github.io/sb_network_timezones/network_timezones.txt' Try disabling Cert Verification on the advanced tab of /config/general
09:13:23 WARNING::MAIN :: Updating network timezones failed, this can happen from time to time. URL: http://sickchill.github.io/sb_network_timezones/network_timezones.txt
09:13:23 ERROR::ThreadPoolExecutor-0_0 :: [51328d1] Failed doing webui callback: Traceback (most recent call last):
  File "/Applications/SickRage/sickbeard/webserve.py", line 287, in async_call
  File "/Applications/SickRage/sickbeard/webserve.py", line 1299, in update
    # do a hard restart
  File "/Applications/SickRage/sickbeard/webserve.py", line 692, in _genericMessage
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'
Traceback (most recent call last):
  File "/Applications/SickRage/sickbeard/webserve.py", line 287, in async_call
    
  File "/Applications/SickRage/sickbeard/webserve.py", line 1299, in update
    # do a hard restart
  File "/Applications/SickRage/sickbeard/webserve.py", line 692, in _genericMessage
    
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'
09:13:23 ERROR::Uncaught exception in write_error
Traceback (most recent call last):
  File "/Applications/SickRage/lib/tornado/web.py", line 1036, in send_error
    self.write_error(status_code, **kwargs)
  File "/Applications/SickRage/sickbeard/webserve.py", line 192, in write_error
    t = PageTemplate(rh=self, filename="404.mako")
  File "/Applications/SickRage/sickbeard/webserve.py", line 165, in render
    kwargs['backtrace'] = RichTraceback()
  File "/Applications/SickRage/lib/mako/template.py", line 471, in render_unicode
    as_unicode=True)
  File "/Applications/SickRage/lib/mako/runtime.py", line 838, in _render
    **_kwargs_for_callable(callable_, data))
  File "/Applications/SickRage/lib/mako/runtime.py", line 873, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/Applications/SickRage/lib/mako/runtime.py", line 899, in _exec_template
    callable_(context, *args, **kwargs)
  File "/Applications/SickRage/cache/mako/layouts/main.mako.py", line 123, in render_body
    __M_writer(unicode(sickbeard.SICKRAGE_BACKGROUND))
AttributeError: 'module' object has no attribute 'SICKRAGE_BACKGROUND'

After running this, the install is unusable and must be re-cloned and start over. As it keeps trying to update itself.

On top of this i am getting SSL errors like listed below.  I have updated open SSL so why its saying that I have no clue.  

Current SSL:

TJ-Johnsens-Mac-Pro:~ tjenvy$ openssl version -a
OpenSSL 1.0.2p  14 Aug 2018

Errors:

09:03:33 INFO::MAIN :: SSL Error requesting url: 'https://sickchill.github.io/sb_network_timezones/network_timezones.txt' You have OpenSSL 0.9.8zh 14 Jan 2016, try upgrading OpenSSL to 1.0.1e+

Now this is a Fresh zip from GitHub, but unzipped and ran, I get this:::

Branch: master
Commit: 51328d1fb61ebc9704934e3cf14425453196790d
Version: v2018.09.17-1
Database Version: 44.0

How am I getting a stale copy direct form GitHub?  Something seems off.

So to summarize:

1.  Open SSL errors that are not true
2.  Unable to update
3.  New zip form GitHub, is the wrong version

**Comment by tjenvy** [comment]
ummmm  IDK why in super bold print are the words  "do a hard restart" but thats not in the issue request at all, I am not sure where that came from.
